client/Lykke.Service.Assets.Client/ReadModels/AssetPairsReadModel.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Autofac;
+using Lykke.Service.Assets.Client.Models.v3;
+using Microsoft.Extensions.Caching.Memory;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Lykke.Service.Assets.Client.ReadModels
+{
+    class AssetPairsReadModel : IAssetPairsReadModel, IStartable
+    {
+        public const string AllKey = "AllAssetPairs";
+
+        private readonly IAssetsService _assetsService;
+        private readonly IMemoryCache _cache;
+
+        public AssetPairsReadModel(IAssetsService assetsService, IMemoryCache cache)
+        {
+            _assetsService = assetsService;
+            _cache = cache;
+        }
+
+        public AssetPair Get(string id)
```

Comment:
Consider to rename the moethid to TryGet to express, that it could return null

Reply:
TryGet should return bool and have "out" in signature. I would prefer to use current syntax.

Reply:
As for me, the major thing here is to show that result of the method coule be invalid, e.g ```null```. There is necessety no follow exactly bool TryXXX(..., out) Such signature used in framework type because out type there could be a value type. But meaning still the same - the method could return invalid result.